src/game/common/rts/specialpower.h

```diff
@@ -65,8 +77,15 @@ class SpecialPowerStore : public SubsystemInterface
     virtual void Update() override {}
 
     bool Can_Use_Special_Power(Object *obj, const SpecialPowerTemplate *special_power_template);
+    SpecialPowerTemplate *Find_Special_Power_Template_Private(Utf8String name);
+    const SpecialPowerTemplate *Find_Special_Power_Template(Utf8String name);
```

Comment:
This function and 2 below can be const.

Reply:
Original is not const. Want to keep things matching original at this point (obviously as the project progresses, we can deviate from original in these areas but I see no benefit to doing it now.)